docs/examples/safe-remote.rst

```diff
@@ -97,12 +118,24 @@ Safe Remote Purchase
             // It is important to change the state first because
             // otherwise, the contracts called using `send` below
             // can call in again here.
-            state = State.Inactive;
-
-            // NOTE: This actually allows both the buyer and the seller to
-            // block the refund - the withdraw pattern should be used.
+            state = State.Release;
 
             buyer.transfer(value);
-            seller.transfer(address(this).balance);
+        }
+
+        /// This function refunds the seller, i.e.
+        /// pays back the locked funds of the seller.
+        function refundSeller()
+            public
+            onlySeller
+            inState(State.Release)
+        {
+            emit SellerRefunded();
+            // It is important to change the state first because
+            // otherwise, the contracts called using `send` below
+            // can call in again here.
+            state = State.Inactive;
+
+            seller.transfer(3 * value);
```

Comment:
```suggestion
            seller.transfer(2 * value);
```
Since the introduction talks about `The following example, both parties have to put twice the value of the item into the contract as escrow.`

Reply:
Please ignore ;)